### ISSUE TITLE
the code displays the not good results unlike the paper

### ISSUE THREAD

**Issue by Harrison-cc** [Alibaba-NLP/HiAGM#13]
INFO:  TEST performance at epoch 155 --- Precision: 0.320000, Recall: 0.285714, Micro-F1: 0.301887, Macro-F1: 0.032500, Loss: 0.396801.
INFO:  TEST performance at epoch 134 --- Precision: 0.428571, Recall: 0.321429, Micro-F1: 0.367347, Macro-F1: 0.052500, Loss: 0.408921.

**Comment by JianhuanZhuo** [comment]
Test on the original code without big changes
The result is:

WARNING:  Performance has not been improved for 50 epochs, updating learning rate
WARNING:  Learning rate update 0.0001--->0.0001
WARNING:  Performance has not been improved for 50 epochs, stopping train with early stopping
100%|██████████| 1/1 [00:00<00:00,  1.36it/s]
INFO:  TEST performance at epoch 73 --- Precision: 0.750000, Recall: 0.214286, Micro-F1: 0.333333, Macro-F1: 0.021429, Loss: 0.418117.

100%|██████████| 1/1 [00:00<00:00,  1.34it/s]
INFO:  TEST performance at epoch 155 --- Precision: 0.391304, Recall: 0.321429, Micro-F1: 0.352941, Macro-F1: 0.046786, Loss: 0.385901.

Any help?

**Comment by gunquxuexia** [comment]
> 信息：第 155 纪元的测试性能 --- 精度：0.320000，召回率：0.285714，Micro-F1：0.301887，Macro-F1：0.032500，损失：0.396801。 信息：第 134 纪元的测试性能 --- 精度：0.428571，召回率：0.321429，Micro-F1：0.367347，Macro-F1：0.052500，损失：0.408921。

请问一下您是怎么操作的呢，我代码下载下来之后就束手无策了，新手小白
